docs/source/intervention_models/multiple_myeloma_treatment/dara_coverage_from_flatiron.csv

```diff
@@ -0,0 +1,66 @@
+Date,Line,Dara % (all),Dara % (Isa corresponding)
+2016/01/01,1.0,0.44%,0.29%
+2016/01/01,2.0,6.64%,6.53%
```

Comment:
Why have this as `%` instead of max of 1?

Reply:
Because I was trying to find a middle ground between machine- and human-readability. But given Abie's ask for graphs above, I think I will make this a decimal proportion and not display the whole table in the docs.